src/main/java/com/ea/restaurant/entities/LoginClientResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.ea.restaurant.entities;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode
+@Setter
+@Getter
+@Builder
+public class LoginClientResponse {
```

Comment:
This is not an entity, you can move it to a new folder `/dtos` in the top of the root project folder at the same level of `/entities`

Reply:
Also you can rename it to `LoginResponseDto`

Reply:
Done.